src/spacefinder/liveblog-adverts.ts

```diff
@@ -147,24 +145,60 @@ const stopListening = () => {
 	document.removeEventListener('liveblog:blocks-updated', onUpdate);
 };
 
-const fill = (rules: SpacefinderRules) => {
-	const options: SpacefinderOptions = { pass: 'inline1' };
+const getPreviousContentBlock = (element: HTMLElement): HTMLElement | null => {
+	const prevElement = element.previousSibling;
+	if (prevElement === null) return null;
```

Comment:
This check would ensure `prevElement` isn't `null` and is a `HTMLElement`, making the assertions that follow unnecessary.

```suggestion
	if (!(prevElement instanceof HTMLElement)) return null;
```